scheduler/src/cook/test/testutil.clj

```diff
@@ -38,7 +38,8 @@
             [qbits.jet.server :refer (run-jetty)]
             [ring.middleware.params :refer (wrap-params)]
             [cook.scheduler.scheduler :as sched]
-            [cook.mesos.task :as task])
+            [cook.mesos.task :as task]
+            [cook.config :as config])
```

Comment:
Let's keep these lexicographically ordered.

Reply:
i think this `require` isn't being used, so I am taking it out